### PR TITLE
#184759954 update password

### PR DESCRIPTION
**Pull request by Fidela1**
### What does this PR do?

This PR implements update password

### Description of Task to be completed?

- User should be able to update password 

### How should this be manually tested?

1.Start [Vikings backend](https://github.com/atlp-rwanda/vikings-ec-bn.git)
2. Clone [repo](https://github.com/atlp-rwanda/vikings-ec-fe.git)
3. Change into the directory `vikings-ec-fe`
4. Checkout to the branch: `feature/update-password-184759954`
5. Run the command `npm install`
6. Run the command `npm run start`
7. Login
8. Click on profile 
9. Click on security and update password

### What are the relevant pivotal tracker stories?
[184759954](https://www.pivotaltracker.com/story/show/184759954)
#### Screenshots (if appropriate)
Figma
Screen Shot 2023-04-18 at 17 43 38

Screen Shot 2023-04-18 at 18 14 16

### CODE REVIEW

**Review by irakozetony — APPROVED** [review]

**Review by fabmukunzi — APPROVED** [review]

File: src/layout/ProfileLayout.js [hunk 1]
```diff
@@ -11,9 +11,10 @@ const ProfileLayout = () => {
   const isAuthenticated = useSelector((state) => state.login.isAuthenticated) === true ? useSelector((state) => state.login.isAuthenticated) : useSelector((state) => state.twoFactorAuth.isAuthenticated);
   const navigate = useNavigate();
   useEffect(() => {
+  console.log('------------------------------------')
```
Comment (ncutixavier) [inline src/layout/ProfileLayout.js]: Remove all logs and unnecessary comments

File: src/pages/auth/UpdatePasswordPage.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      <UpdatePasswordForm/>
+      </div>
+     </div>
+    
+  )
+}
+
+export default updatePasswordPage;
+
+
```
Comment (ncutixavier) [inline src/pages/auth/UpdatePasswordPage.js]: `<UpdatePasswordForm/>` was not necessary, Put all the code inside this page.

File: src/validations/index.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import * as yup from 'yup';
+
+export const loginSchema = yup.object().shape({
+  oldPassword: yup
+    .string()
+    .min(8, 'Password must be at least 8 characters long')
+    .required('Password is required'),
+});
```
Comment (ncutixavier) [inline src/validations/index.js]: Why did you create a login schema? Are you working on the login?

**Review by ncutixavier — CHANGES_REQUESTED** [review]
Remember to squash your commit

**Review by Aline-Uwera — APPROVED** [review]

File: src/pages/auth/UpdatePasswordPage.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+      showSuccessMessage(response.message);
+    } catch (error) {
+      showErrorMessage(error.response.data.message);
+    }
+  };
+  return (
+      <div className="flex flex-col ">
+       <div className="hidden sm:flex ">
+       </div>
+       {/* <div className=" md:w-3/5 px-3 border border-grey-400 p-3 flex-grow"> */}
```
Comment (ncutixavier) [inline src/pages/auth/UpdatePasswordPage.js]: Remove all unnecessary comments

**Review by ncutixavier — COMMENTED** [review]
- Please update your PR description by specifying all steps to test your task

**Pull request merged**